pkg/app/impl/authz.go

```diff
@@ -55,10 +58,18 @@ func NewAuthorizerServer(
 ) *AuthorizerServer {
 	newLogger := logger.With().Str("component", "api.grpc").Logger()
 
+	const certs = `https://citadel.demo.aserto.com/dex/keys`
```

Comment:
Why do we need this as a hardcoded constant string?

Reply:
This was a leftover from my experiments, removed it.